options.go

```diff
@@ -177,6 +183,25 @@ func IdentityResolverFunc(resolver IdentityResolver) Option {
 	}
 }
 
+// Installs RingChecksumStatPeriod, which determines the period between
```

Comment:
ack, and thanks for the info. @dansim ought we include linting in our pre-commit hooks?